nva-datamodel-java/src/main/java/no/unit/nva/api/PublicationResponseElevatedUser.java

```diff
@@ -37,6 +39,14 @@ public static PublicationResponseElevatedUser fromPublication(Publication public
         response.setAdditionalIdentifiers(publication.getAdditionalIdentifiers());
         response.setRightsHolder(publication.getRightsHolder());
         response.setPublicationNotes(publication.getPublicationNotes());
+        response.setAllowedOperations(Set.of());
+        return response;
+    }
+
+    public static PublicationResponseElevatedUser fromPublicationWithAllowedOperations(Publication publication,
+                                                                           Set<String> allowedOperations) {
```

Comment:
Honestly, I'd prefer that allowedOperations to be an Set of enum

Reply:
If changing to enum, should probably align with publication-api https://github.com/BIBSYSDEV/nva-publication-api/blob/c1ac089ea327b5a3fb63850d174f1d4222288a04/publication-commons/src/main/java/no/unit/nva/publication/permission/strategy/PublicationAction.java